dictionary.rb

```diff
@@ -0,0 +1,16 @@
+class Dictionary
+  attr_reader :dictionary
+
+  def initialize
+    @dictionary = File.read('google-10000-english-no-swears.txt')
+  end
+
+  def trim_dictionary
+    result = []
+    words = @dictionary.split("\n")
+    words.each do |word|
```

Comment:
Remember what I told you about running loops in which you update value from outside? That's the case right here.

Please use `#filter`.

Also, your methods should be named after what it does, or what it returns. I would rename this one to `#words`.

Judged by how you use this class, it seems unnecessary, it could just be

```
@dictionary = File.read('google-10000-english-no-swears.txt').split("\n").filter{|word| word.length >= 5 && word.length <= 12}
```
In the `Game` class, no?